README.md

```diff
@@ -132,8 +126,8 @@ Function that can be used to obtain popper reference
 placement?: PopperJS$Placement;
 ```
 
-One of the accepted placement values listed in the [Popper.js documentation](https://popper.js.org/popper-documentation.html#Popper.placements).  
-Your popper is going to be placed according to the value of this property.  
```

Comment:
2 trailing whitespaces in markdown mean "newline" (`<br />` tag), please may you revert this?